### ISSUE TITLE
Why does my model throw an exception when it contains nn.ConvTranspose3d(deconvolution)?

### ISSUE THREAD

**Issue by huang229** [microsoft/onnxruntime#2722]
I use Pytorch1.2.0 to build the model. I test the exported model on the C++ side. I tested my model countless times. The session. Run() function throws an exception whenever the model contains nn.ConvTranspose3d. But I can use torch.onnx.export () to export the model normally on the python side.

The code of the exported model is as follows:
class dconv_leaky(nn.Module):
        def __init__(self, in_channels, out_channels, kernel_size, stride):
            super(dconv_leaky, self).__init__()
            self.deconv = nn.ConvTranspose3d(in_channels, out_channels, kernel_size=kernel_size, stride=stride, bias=False)
            self.bn1 = nn.BatchNorm3d(out_channels)
            self.relu = nn.LeakyReLU(0.1, inplace=True)
            nn.init.xavier_normal_(self.deconv.weight.data)
            # nn.init.constant_(self.deconv.bias.data, 0.0)
            self.bn1.weight.data.fill_(1)
            self.bn1.bias.data.zero_()
        def forward(self, x):
            x = self.deconv(x)
            x = self.bn1(x)
            x = self.relu(x)
            return x

class testnet(nn.Module):
    def __init__(self):
        super(testnet, self).__init__()
        self.conv1 = dconv_leaky(64, 64, 2, 2)
        self.conv2 = dconv_leaky(64, 64, 2, 2)
    def forward(self, data1):
        conv1 = self.conv1(data1)
        conv2 = self.conv2(data1)
        return torch.cat([conv1, conv2], dim=1)

model = testnet()
dummy_input1 = torch.randn(1, 64, 8, 4, 4, device='cpu')
out = model(dummy_input1)
input_names1 = "input_1"
output_names = ["output1"]
torch.onnx.export(model, (dummy_input1), "tooth_seg.onnx",verbose=True,input_names=[input_names1], output_names=output_names)
  %13 : Float(1, 64, 16, 8, 8) = onnx::ConvTranspose[dilations=[1, 1, 1], group=1, kernel_shape=[2, 2, 2], pads=[0, 0, 0, 0, 0, 0], strides=[2, 2, 2]](%input_1, %conv1.deconv.weight), scope: testnet/dconv_leaky[conv1]/ConvTranspose3d[deconv] # D:\python-3.6.7\lib\site-packages\torch\nn\modules\conv.py:930:0
  %14 : Float(1, 64, 16, 8, 8) = onnx::BatchNormalization[epsilon=1e-05, momentum=0.9](%13, %conv1.bn1.weight, %conv1.bn1.bias, %conv1.bn1.running_mean, %conv1.bn1.running_var), scope: testnet/dconv_leaky[conv1]/BatchNorm3d[bn1] # D:\python-3.6.7\lib\site-packages\torch\nn\functional.py:1656:0
  %15 : Float(1, 64, 16, 8, 8) = onnx::LeakyRelu[alpha=0.1](%14), scope: testnet/dconv_leaky[conv1]/LeakyReLU[relu] # D:\python-3.6.7\lib\site-packages\torch\nn\functional.py:1060:0
  %16 : Float(1, 64, 16, 8, 8) = onnx::ConvTranspose[dilations=[1, 1, 1], group=1, kernel_shape=[2, 2, 2], pads=[0, 0, 0, 0, 0, 0], strides=[2, 2, 2]](%input_1, %conv2.deconv.weight), scope: testnet/dconv_leaky[conv2]/ConvTranspose3d[deconv] # D:\python-3.6.7\lib\site-packages\torch\nn\modules\conv.py:930:0
  %17 : Float(1, 64, 16, 8, 8) = onnx::BatchNormalization[epsilon=1e-05, momentum=0.9](%16, %conv2.bn1.weight, %conv2.bn1.bias, %conv2.bn1.running_mean, %conv2.bn1.running_var), scope: testnet/dconv_leaky[conv2]/BatchNorm3d[bn1] # D:\python-3.6.7\lib\site-packages\torch\nn\functional.py:1656:0
  %18 : Float(1, 64, 16, 8, 8) = onnx::LeakyRelu[alpha=0.1](%17), scope: testnet/dconv_leaky[conv2]/LeakyReLU[relu] # D:\python-3.6.7\lib\site-packages\torch\nn\functional.py:1060:0
  %output1 : Float(1, 128, 16, 8, 8) = onnx::Concat[axis=1](%15, %18), scope: testnet # D:/tensorflow-project/pytorch-test/onnx_test.py:54:0
  return (%output1)

The code to throw an exception at the C++ end is as follows:
namespace Ort {
inline void ThrowOnError(const OrtApi& ort, OrtStatus* status) {
  if (status) {
    std::string error_message = ort.GetErrorMessage(status);
    OrtErrorCode error_code = ort.GetErrorCode(status);
    ort.ReleaseStatus(status);
    throw Ort::Exception(std::move(error_message), error_code);
  }
}
｝

**Comment by huang229** [comment]
In addition, nn.Convtranspose2d() works normally without any problem.

**Comment by snnn** [comment]
You're right

**Comment by huang229** [comment]
@snnn Looking at the source code, I found that the support for 3D is poor, and many 3D operations have not been implemented. I look forward to adding some commonly used 3D ops. Pray!

**Comment by faxu** [comment]
This is a dupe of https://github.com/microsoft/onnxruntime/issues/965

**Comment by CSGO01** [comment]
Have you solved it?
